### PR TITLE
Update prompt to add support for playlist voice commands

### PR DESCRIPTION
**Pull request by nikito**
Adding additional info to the prompt to teach the LLM how to handle requests to play playlists.

### CODE REVIEW

**Comment by OzGav** [comment]
@jozefKruszynski thoughts?

**Comment by jozefKruszynski** [comment]
Looks pretty reasonable. I assume it's been tested by @nikito and works for him with no code changes.
We could probably also add another playlist slot to support the LLM flow also.

**Comment by nikito** [comment]
> Looks pretty reasonable. I assume it's been tested by @nikito and works for him with no code changes. We could probably also add another playlist slot to support the LLM flow also.

Yes, tested with a few different playlists of mine. 🙂

File: prompt/prompt.txt [hunk 1]
```diff
@@ -5,27 +5,33 @@ Here is the structured JSON that I expect in the first response type {"media_id"
...
+A playlist like this {"media_id":"All Favorited Tracks", "media_type":"playlist"}
 
 You must reply with only the JSON model, nothing before nor after because your response will be processed by a search component of a media listening service.
 
 Here are a few examples of input with the expected output:
 "Hells Bells by ACDC" >> {"media_id":"AC/DC - Hells Bells",  "media_type":"track"}.
 "The artist that composed the soundtrack of Inception" >> {"media_id": "Hans Zimmer", "media_type":"artist"}.
 "a list of 5 classic 80's rock tracks" >>  {"media_id": ["Guns N' Roses - Sweet Child o' Mine", "AC/DC - Back in Black", "Bon Jovi - Livin' on a Prayer", "Def Leppard - Pour Some Sugar on Me", "Journey - Don't Stop Believin'"], "media_type":"track"}
+"Play 500 random tracks playlist" >> {"media_id":"500 Random Tracks", "media_type":"playlist"}
+"Play my liked songs playlist" >> {"media_id":"All Favorited Tracks", "media_type":"playlist"}
```
Comment (marcelveldt) [inline prompt/prompt.txt:37]: This is going to cause issues with the "Liked songs" playlist that exists on Spotify (and maybe Deezer or others as well)

Comment (marcelveldt) [inline prompt/prompt.txt:37]:
```suggestion
"Play my liked songs playlist" >> {"media_id":"Liked songs", "media_type":"playlist"}
"Play my favorited tracks" >> {"media_id":"All Favorited Tracks", "media_type":"playlist"}
```

**Comment by nikito** [comment]
Seems to be working fine with the suggested changes.

**Review by marcelveldt — APPROVED** [review]
Nice, thanks!

**Pull request merged**